alibi/api/interfaces.py

```diff
@@ -64,7 +66,7 @@ class Explainer(abc.ABC):
     """
     Base class for explainer algorithms
     """
-
+    _version: ClassVar = __version__
```

Comment:
Is one meant to set the underling Type of the ClassVar? e.g. ClassVar[str] ?

Reply:
Good catch, it should be `ClassVar[str]`